multiqc/utils/search_patterns.yaml

```
@@ -485,6 +485,9 @@ mapdamage:
   - fn: "3p*_freq.txt"
   - fn: "5p*_freq.txt"
   - fn: "lgdistribution.txt"
+megahit:
+  contents: " - MEGAHIT v"
+  num_lines: 1
```

Just in case there's some random stdout from slurm or something..

```suggestion
  num_lines: 5
```